### ISSUE TITLE
Is MAC M1 supported for CTranslate2 installation ? 

### ISSUE THREAD

**Issue by harishankar-gopalan** [OpenNMT/CTranslate2#724]
I tried installing CTranslate2 on MAC M1 but there doesnt seem to be an available package in pip repository. Do I need to recompile from source or is it not supported yet ?

**Comment by guillaumekln** [comment]
Currently there are no prebuilt packages for Mac M1. You should compile the library from sources.

However, I have an experimental branch building ARM packages for macOS. Can you help testing? You can go to this [build page](https://github.com/guillaumekln/CTranslate2/actions/runs/1944381386), download the artifact "python-wheels", and install appropriate "arm64" wheel.

**Comment by guillaumekln** [comment]
@harishankar-gopalan Did you have the time to install one of the macOS ARM64 wheel from the archive?

**Comment by harishankar-gopalan** [comment]
Hi @guillaumekln I was able to install the manylinux wheel after renaming it to the supported target name of Mac. I havent tested the above yet. Held up a bit in other priority works. Will surely update this thread once I test it out.

**Comment by harishankar-gopalan** [comment]
Hi @guillaumekln  I was able to install the macOS ARM64 wheel from the archive you had listed previously in this thread. Also I did do a basic testing of a single batch of translations. It does work fine.
I guess a basic sanity testing is working. Just to note, I havent done a full blown latency/memory consumption kind of testing the macOS support is more of a development time requirement and not a production deployment.
